pontoon/base/admin.py

```diff
@@ -55,6 +59,44 @@ def save_model(self, request, obj, form, change):
 
             log_user_groups(request.user, obj, (add_groups, remove_groups))
 
+    def delete_model(self, request, obj):
```

Comment:
It would be nice to add a comment here explaining what we're doing here and why (creating a new user to replace the deleted user in order to change the ID).